router/middleware/repo/repo_test.go

```diff
@@ -147,7 +147,7 @@ func TestRepo_Establish_NoRepoParameter(t *testing.T) {
 	// run test
 	engine.ServeHTTP(context.Writer, context.Request)
 
-	if resp.Code != http.StatusBadRequest {
+	if resp.Code == http.StatusOK {
 		t.Errorf("Establish returned %v, want %v", resp.Code, http.StatusBadRequest)
 	}
```

Comment:
Did we need to update this test? I would think the server is expecting StatusBadRequest so the former was accurate. 
Maybe Im missing something, so we could instead update the ErrorF to reflect the change?

Reply:
Removing the repo param now searches for builds instead of throws an error. I updated the test to reflect this.